go.mod

```diff
@@ -1,14 +1,15 @@
 module github.com/tektoncd/results
 
-go 1.23
+go 1.23.0
```

Comment:
I have to make it `1.23.0` because when I run `go mod tidy` with 1.23 it is making it to `go 1.23.0` and also adding `gotoolchain 1.23.6`

@vdemeester do you have any idea on how we can fix this?

Reply:
Why do you want to "fix" this? Leave it like this (1.23.0).

Reply:
I think somewhere it was discussed we prefer to use 1.23 and not 1.23.0
So I am thinking how can I do this.

Reply:
Interesting. Why? Last time I checked few other Tekton projects. They were all using vX.Y.Z versioning, so why not keep it common and change once we start changing the other projects? My 2c.

Reply:
Yeah, so when we want to have version like `1.23.4` then format `x.y.z` is preferred but when it is `x.y.0` then `x.y` is preferred. This is what my understanding is from [this](https://github.com/tektoncd/plumbing/pull/2477#discussion_r1973292136) discussion. Please correct, if I am wrong.

Reply:
But dependable or go mod tidy will put it back to 1.23.0, isn't it? So I don't think we should use 1.23.

Reply:
yeah